.github/workflows/pages-deploy.yml.hook

```diff
@@ -22,50 +29,9 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Bundle Caching
-        id: bundle-cache
-        uses: actions/cache@v1
-        with:
-          path: vendor/bundle
-          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile') }}
-          restore-keys: |
-            ${{ runner.os }}-gems-
-
-      - name: Bundle config
-        run: |
-          bundle config path vendor/bundle
-
-      - name: Bundle Install
-        if: steps.bundle-cache.outputs.cache-hit != 'true'
+      - name: install tooling
         run: |
-          bundle install
-
-      - name: Bundle Install locally
-        if: steps.bundle-cache.outputs.cache-hit == 'true'
-        run: |
-          bundle install --local
-
-      - name: Build Site
-        run: |
-          bash tools/build.sh -b ""
-
-      - name: Test Site
-        run: |
-          bash tools/test.sh
-
-  deploy:
-    needs: build-n-test
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/setup-ruby@v1
-        with:
-          ruby-version: '2.6.x'
-
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
+          brew install yq
```

Comment:
Ubuntu did not have the `brew` tool

Reply:
It has. :)

Reply:
Oh, I used to think the `brew` was only available on macOS 😂

Reply:
Yeah me too, until I got a PC recently and installed Linux. :)